pages/common/az-webapp.md

```diff
@@ -0,0 +1,21 @@
+# az webapp
+
+> Manage Web Applications hosted in Azure Cloud Services.
+> Part of `azure-cli`.
+> Mode information: <https://learn.microsoft.com/cli/azure/webapp>.
+
+- List available runtimes for a webapp:
+
+`az webapp list-runtimes --os-type {{windows|linux}}`
+
+- Create a web app:
+
+`az webapp up --name {{app_name}} --location {{location}} --runtime {{runtime}}`
```

Comment:
We try to avoid  abbreviations, or `name` or `application_name`
```suggestion
`az webapp up --name {{name}} --location {{location}} --runtime {{runtime}}`
```